ministryofjustice/hmpps-probation-integration-e2e-tests
# PI-2524 Test fixes

* Use local date format for Prison API
* Remove CAS2 "risk factors" section
* Update MMS landing page title
* Update Pathfinder date format
* Update AutomatedTestUser email address
* Update "Work after release" link text
* Add R&M "Further information" field

Note: there are still a few more failures to resolve around OASys and CVL - see https://github.com/ministryofjustice/hmpps-probation-integration-e2e-tests/actions/runs/11220741263
